api/workAllocation/util.ts

```diff
@@ -366,7 +366,8 @@ export function prepareRoleApiRequest(jurisdictions: string[], locationId?: numb
     // TODO: This should not be hard-coded list
```

Comment:
Can we move it to default.json and custom-environment-variables.json. You'll have to do a split on a comma separated value to get the array required here. I'm OK if you raise a separate tech debt ticket for this